ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/multiplechoice/MultipleChoiceTestTags.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.ui.datacollection.tasks.multiplechoice
+
+object MultipleChoiceTestTags {
```

Comment:
we don't need object, right?
we can keep constants as is

Reply:
IMO, wrapping this under object makes it easier to use the constants

Reply:
easier to use but object MultipleChoiceTestTags will create a extra memory, object creation

in terms of using, we can directly write `MULTIPLE_CHOICE_LIST` no object name reference required

Reply:
`MultipleChoiceTestTags` is a singleton, so the memory for the object reference is minimal, if not zero if optimized out by the compiler.

Reply:
According to Gemini, top-level declarations are preferred in Kotlin, but haven't validated this yet:

<html><head></head><body><div><div jsname="OsZLCd" data-mrc="GAE" class="yf" data-async-fc="EqEBCmJBSkc5SmtOV2JlVHIwWlJsb0g1UlhZNFhGQkdzcFRrUG9jN2k2Nk9WemV3aVpSRGNQUVY0RU9LTkt4dzNRbVNLWGI3Qm44ekdKOFplLUtJblJ5dkFGWmdKbjAzUF9QZEN6URIXcElSbFo0QzBLUEt1NU5vUDN0WEN3QTAaIkFGWHJFY3IyNU9xZXNpX0hlSVlZQTF4cFdYOVZEOE1tMVE" data-async-fcv="3" data-async-ons="7159" id="fc_pIRlZ4C0KPKu5NoP3tXCwA0_1" eid="54RlZ_GDBuCliLMPyvqCQQ"><div jsname="OsZLCd" data-mdv="0" data-mrc="GAE" class="yp" data-async-fc="EqABCmJBSkc5SmtQcU0wU2dIWmpjZTU0SE9fNHM0TGRFdTdyMXpVYzFmbXZuNmliMnlDWDRFeTdtVmRWbmNDZ25vSzNLQUlEaDh4WFhva0JfbUx3dHNyYlJqaFBDd2JHM3VkaDNZQRIWNTRSbFpfR0RCdUNsaUxNUHl2cUNRURoiQUZYckVjcHdJYVFLeC1md3l5UlhGWm41dlhBdFlna09Mdw" data-async-fcv="3" data-async-ons="7159" id="fc_54RlZ_GDBuCliLMPyvqCQQ_1"><div><div jsname="GVaozf" style="font-size:11pt"><div data-di="0"><div><span>While you can technically use a Kotlin object as a namespace, it's generally not the recommended approach. </span><span>Here's why: </span></div><br style="line-height:11pt"><div style="font-size:13pt;font-weight:bold;margin-bottom:11pt"><span>Why not to use objects as namespaces: </span></div><ul><li><span><span style="font-weight:bold">Top-level declarations are preferred:</span> </span><span>Kotlin encourages the use of top-level functions and properties, which naturally provide a way to organize code into logical units without creating unnecessary objects. </span></li><li><span><span style="font-weight:bold">Readability and maintainability:</span> </span><span>Using objects as namespaces can make your code harder to read and understand. </span><span>Top-level declarations are more straightforward and easier to locate. </span></li><li><span><span style="font-weight:bold">File facade issues (JVM):</span> </span><span>On the JVM, Kotlin compiles top-level declarations into a file facade class. </span><span>If you use objects as namespaces, you might encounter naming conflicts with these generated classes. </span></li></ul><div style="font-size:13pt;font-weight:bold;margin-bottom:11pt"><span>Recommended alternatives: </span></div><ul><li><span><span style="font-weight:bold">Top-level functions and properties:</span> </span><span>The preferred way to group related code is to use top-level functions and properties within the same file. </span></li><li><span><span style="font-weight:bold">Packages:</span> </span><span>For larger projects, use packages to organize your code into meaningful modules. </span></li></ul><div><span>Example of using top-level declarations: </span></div><br style="line-height:11pt"><div data-cbl="" data-cbc="">

Reply:
Thanks for the detailed explaination! I've moved the consts to top-level properties within the same file.